### PR TITLE
Requests on the homepage 

### PR DESCRIPTION
**Pull request by Gianfranco97**
### Changes description

In this PR:
- Create the public URL correctly 
- Verify session validity

### Checklist

Please check if your PR fulfills the following specifications:

- [ ] Tests for the changes have been added
- [ ] Docs have been added/updated

### References

Close: #700

### CODE REVIEW

File: src/shared/handleMessage.js [hunk 1]
```diff
@@ -1,7 +1,5 @@
-import history from './history'
-import glpi from './glpiApi'
-import publicURL from './publicURL'
 import { I18n } from "react-i18nify"
+import logout from './publicURL'
```
Comment (hectorerb) [inline src/shared/handleMessage.js]: why you import logout from publicURL?

File: src/shared/publicURL.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 const path = () => {
     var location = document.createElement("a")
     location.href = process.env.PUBLIC_URL
-    return location.pathname
+    return (process.env.PUBLIC_URL !== '') ? location.pathname : ''
```
Comment (hectorerb) [inline src/shared/publicURL.js]: Is this validation really necessary?

Comment (Gianfranco97) [reply src/shared/publicURL.js]: yes, otherwise the path is '/' and it generates errors

File: src/shared/handleMessage.js (hunk 1 above)
Comment (Gianfranco97) [reply src/shared/handleMessage.js]: I'm sorry, my mistake

**Pull request merged**